packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/CustodialTx/model.tsx

```diff
@@ -152,18 +152,21 @@ export const Origin = (props: Props) => {
     case 'REFUNDED':
     case 'DEPOSIT':
       return props.tx.amount.symbol in CoinTypeEnum ? (
-        <>{props.coinTicker} Wallet</>
+        <>{props.supportedCoins[props.coin].displayName} Wallet</>
       ) : (
         <>Bank Account</>
       )
     case 'SELL':
       return props.tx.extraAttributes?.direction === 'FROM_USERKEY' ? (
-        <>{props.tx.amount.symbol} Wallet</>
+        <>{props.supportedCoins[props.tx.amount.symbol].displayName} Wallet</>
```

Comment:
I guess this should be fine if typscript didn't complain about possible undefined keys here, but doing this many dot notations is a little scary. Maybe useful to use Ramda `path` or `defaultTo` just in case one of those is undefined.